storage/store.go

```diff
@@ -1324,8 +1319,7 @@ func (s *Store) ProposeRaftCommand(idKey cmdIDKey, cmd proto.InternalRaftCommand
 		panic("proposed a nil command")
 	}
 	// Lazily create group. TODO(bdarnell): make this non-lazy
-	err := s.multiraft.CreateGroup(uint64(cmd.RaftID))
-	if err != nil {
+	if err := s.multiraft.CreateGroup(uint64(cmd.RaftID)); err != nil {
```

Comment:
I'm not going to argue for one or the other, but my understanding is that this is a change based on your personal preference for less local scope in exchange for longer lines. I generally prefer to leave things alone unless you feel strongly about something. Correct me if you think otherwise, but this seems like a nit.